src/features/calendar/components/EventCluster/Event.tsx

```diff
@@ -162,8 +168,15 @@ const Event = ({
     <Box className={classes.container}>
       {topBadge}
       {collapsed && (
-        <Box className={classes.collapsedContainer}>
-          <Typography className={classes.title}>{title}</Typography>
+        <Box
+          className={classes.collapsedContainer}
+          onMouseEnter={() => setIsHovered(true)}
+          onMouseLeave={() => setIsHovered(false)}
```

Comment:
Can these be moved up one level, to the `Box` on line 168, and that way not have to be duplicated again on line 197-198?